src/main/java/org/jabref/gui/maintable/columns/FileColumn.java

```diff
@@ -138,10 +139,20 @@ private void setCommonSettings() {
     }
 
     private String createFileTooltip(List<LinkedFile> linkedFiles) {
-        if (!linkedFiles.isEmpty()) {
-            return Localization.lang("Open file %0", linkedFiles.getFirst().getLink());
+        if (linkedFiles.isEmpty()) {
+            return null;
+        }
+
+        StringJoiner joiner = new StringJoiner("\n");
+
+        for (LinkedFile linkedFile : linkedFiles) {
+            joiner.add(linkedFile.getLink());
+        }
```

Comment:
Can you convert that to stream. At the end `Collectors.joining("\n")`  - even more modern java then.